src/scripts/antithesis.zig

```diff
@@ -159,6 +180,68 @@ fn push_image(shell: *Shell, image: Image, tag: []const u8) !void {
     );
 }
 
+fn trigger_test(
+    shell: *Shell,
+    tag: []const u8,
+    antithesis_user: []const u8,
+    antithesis_password: []const u8,
+) !void {
+    const commit = try shell.exec_stdout("git rev-parse HEAD", .{});
+
+    var body = std.ArrayList(u8).init(shell.gpa);
+    defer body.deinit();
+
+    try std.json.stringify(.{
+        .params = .{
+            .@"custom.duration" = "0.5",
+            .@"antithesis.images" = try shell.fmt("workload:{s};replica:{s}", .{ tag, tag }),
+            .@"antithesis.config_image" = try shell.fmt("config:{s}", .{tag}),
+            .@"antithesis.description" = commit,
+            .@"antithesis.report.recipients" = "oskar@tigerbeetle.com",
+        },
+    }, .{}, body.writer());
+
+    var client = std.http.Client{ .allocator = shell.gpa };
```

Comment:
wow, we have a usable http client in stdlib! I don't think we've used it before, I am wondering if we should replace some of our shelling out with this stuff...